pages/post/[slug].tsx

```diff
@@ -19,33 +20,38 @@ import StandWithUkraine from "@/components/StandWithUkraine";
 const Post: NextPage<{
   post: PostDocument;
   latestPosts: PostDocumentWithoutContent[];
-}> = ({ post, latestPosts }) => (
-  <>
-    <Head>
-      <title>{post.title}</title>
-      <meta name="description" content={config.site_description} />
-      <meta property="og:description" content={config.site_description} />
-      <meta property="og:url" content={config.host_url} />
-      <meta property="og:type" content="website" />
-      <meta property="og:site_name" content={config.site_title} />
-      <meta property="og:image" content={config.defaultImage} />
-      <link rel="icon" href="/favicon.ico" />
-    </Head>
-    <StandWithUkraine />
-    <Header />
-    <BreadCrumbs title={post.title} />
-    <PageProgress />
-    <section className="blogpost-section">
-      <div className="container">
-        <div className="blogpost-outer">
-          <PostContent post={post} />
-          <LatestPosts latestPosts={latestPosts} />
+}> = ({ post, latestPosts }) => {
+  const description = getFirstParagraph(post.content);
+
+  return (
+    <>
+      <Head>
+        <title>{post.title}</title>
+        <meta property="og:title" content={post.title} />
+        <meta name="description" content={config.site_description} />
+        <meta property="og:description" content={description} />
```

Comment:
```suggestion
        <meta property="og:description" content={getFirstParagraph(post.content);} />
```
Run the function inside, so the component stays dumb, as it does not have any state anyway